lib/compiler_rt/README.md

```diff
@@ -95,24 +95,24 @@ Integer and Float Operations
 | ✓ | __mulsi3           | i32  | i32  | i32  | `a * b`                        |
 | ✓ | __muldi3           | i64  | i64  | i64  | ..                             |
 | ✓ | __multi3           | i128 | i128 | i128 | ..                             |
-| ✓ | __divsi3           | i32  | i32  | i32  | `a / b`                        |
+| ✓ | __divsi3           | i32  | i32  | i32  | `a / b` signed                 |
```

Comment:
signed is redundant as you have the types given as `i32` here. dito below.

Reply:
Thanks; I remove both the signed and unsigned comment.

I'm also thinking to add a description/comment to `fix` and `fixuns`:
  - `fix...` | float to int, rounding toward 0. |
  - `fixuns...` | float to uint, rounding toward 0. negative values become 0. |

What so you think?

Reply:
Souds good, since I forgot to add them. Only on the first entry then please.

Reply:
Conversion kinda implies it, but being abit more explicit sounds okayish to me.